serverless-sample/app/azure/views/index.html

```diff
@@ -0,0 +1,185 @@
+                            c-16.9,23.3-39.7,40.4-79.3,40.4C499.8,189.4,458.6,150.2,458.6,96.8z"/>
+                    </g>
+                    <g>
+                        <path d="M659.2,6.7h49.6v135.6h86.8v43.5H659.2V6.7z"/>
+                    </g>
+                    <g>
+                        <path d="M824.6,6.7h69.1c64,0,101.1,36.9,101.1,88.5v0.5c0,51.7-37.6,90.1-102.1,90.1h-68.1V6.7z M894.4,141.8
+                            c29.7,0,49.4-16.4,49.4-45.3V96c0-28.7-19.7-45.3-49.4-45.3h-20.2v91.1H894.4z"/>
+                    </g>
+                    <g>
+                        <path d="M458.6,310v-0.5c0-52.2,40.2-93.2,94.4-93.2c36.6,0,60.1,15.4,76,37.4l-37.4,28.9c-10.2-12.8-22-21-39.2-21
+                            c-25.1,0-42.7,21.2-42.7,47.3v0.5c0,26.9,17.7,47.9,42.7,47.9c18.7,0,29.7-8.7,40.4-21.8l37.4,26.6
+                            c-16.9,23.3-39.7,40.4-79.3,40.4C499.8,402.7,458.6,363.5,458.6,310z"/>
+                        <path d="M553,216.4c36.6,0,60.1,15.4,76,37.4l-37.4,28.9c-10.2-12.8-22-21-39.2-21c-25.1,0-42.7,21.2-42.7,47.3v0.5
+                            c0,26.9,17.7,47.9,42.7,47.9c18.7,0,29.7-8.7,40.4-21.8l37.4,26.6c-16.9,23.3-39.7,40.4-79.3,40.4c-51.2,0-92.4-39.2-92.4-92.6
+                            v-0.5C458.6,257.3,498.8,216.4,553,216.4"/>
+                    </g>
+                    <g>
+                        <path d="M628.3,219.9h55.5l40.7,113.4l40.7-113.4h54.5L747,400.4H701L628.3,219.9z"/>
+                        <path d="M819.7,219.9L747,400.4H701l-72.7-180.4h55.5l40.7,113.4l40.7-113.4H819.7"/>
+                    </g>
+                    <g>
+                        <path d="M830.7,219.9h84.7c27.4,0,46.3,7.2,58.3,19.4c10.5,10.2,15.9,24.1,15.9,41.7v0.5c0,27.4-14.6,45.6-36.9,55l42.7,62.4
+                            h-57.3l-36.1-54.3h-0.5h-21.2v54.3h-49.6V219.9z M913.1,305.9c16.9,0,26.6-8.2,26.6-21.2v-0.5c0-14.1-10.2-21.2-26.9-21.2h-32.5
+                            v43H913.1z"/>
+                        <path d="M915.4,219.9c27.4,0,46.3,7.2,58.3,19.4c10.5,10.2,15.9,24.1,15.9,41.7v0.5c0,27.4-14.6,45.6-36.9,55l42.7,62.4h-57.3
+                            l-36.1-54.3h-0.5h-21.2v54.3h-49.6V219.9H915.4 M880.3,305.9h32.8c16.9,0,26.6-8.2,26.6-21.2v-0.5c0-14.1-10.2-21.2-26.9-21.2
+                            h-32.5V305.9"/>
+                    </g>
+                </g>
+            </g>
+        </svg>
+    </div>
+    <div class="logo"></div>
+    <div class=container>
+        <div class=header>
+            <h1>Congratulations!</h1>
+            <p>If you have reached here everything went well<br>
+                What happened:<br>
+                1. Azure blob container has been created and a borat gif has been uploaded with terraform<br>
+                2. A Golang server has been deployed with an image present in the acr repository, this server serves the uploaded gif on / route of deployed application<br>
+                3. Application has been assigned a cloud run domain and SSL<br>
```

Comment:
need to change this statement azure equivalent cloud service instead of cloud run